templates/php-component/README.md

```diff
@@ -0,0 +1,33 @@
+# my-component
+
+[![Build Status](https://travis-ci.org/keboola/my-component.svg?branch=master)](https://travis-ci.org/keboola/my-component)
```

Comment:
premyslim jestli sem dat odkazy na travis - pro cizi to bude matouci

Reply:
Každá komponenta by se měla buildovat na travisu, ne? Plus generátor automaticky travis nastavuje.

Reply:
cizi lidi to muzou mit jinde nez githubu (bitbucket, gitlab) a pak maji jinej ci tool

Reply:
Můžou, ale návod mají stejně jen pro travis a distrubujeme s tím i .travis.yml, takže bych to nechal.

Reply:
Úplně bokem - jestli se na to nedíváme okem public repozitářů z githubu, který jsou zdarma a je pro ně i zdarma travis. Ne každej to takhle bude chtít.

Reply:
?

Reply:
uz jsem to vyresil https://github.com/keboola/component-generator/issues/8, nechal bych to ted konovi